sdk/core/azure-core/test/ut/proxy_tests/localproxy.passwd/squid.conf

```diff
@@ -0,0 +1,7991 @@
+#	WELCOME TO SQUID 3.5.27
```

Comment:
Other than a couple lines diff, this file is largely identical to the other one (`sdk/core/azure-core/test/ut/proxy_tests/localproxy/squid.conf`). Do we need to check-in both? The files are fairly large.

![image](https://user-images.githubusercontent.com/6527137/192832235-27f1251a-1857-4284-bcdd-5241b3b89f1e.png)

![image](https://user-images.githubusercontent.com/6527137/192832582-c42e6de7-fccd-4350-a155-3329e12e2ba0.png)

Reply:
I struggled with this. The reason all this text is present is because we might need to make other tweaks to the proxy.conf files in the future and this template provides documentation for all the options.

Reply:
Hmm, I see ~2 more such files with minor tweaks to the "template", which might benefit from de-duplicating (if possible).